src/containers/UserProfile/AboutUser.js

```diff
@@ -0,0 +1,156 @@
+import React, { Component } from "react";
+import SimpleReactValidator from "simple-react-validator";
+import ReactNotification from "react-notifications-component";
+import "react-notifications-component/dist/theme.css";
+import addNotification from "../../utilities/index";
+import * as auth from "../../services/Session";
+import API from "../../api/index";
+
+class AboutUser extends Component {
+  constructor(props) {
+    super(props);
+
+    this.validator = new SimpleReactValidator();
+    this.state = {
+      about: this.props.aboutUser
+    };
+    this.handleUserInput = this.handleUserInput.bind(this);
+    this.notificationDOMRef = React.createRef();
+    this.submitForm = this.submitForm.bind(this);
+    this.onCancel = this.onCancel.bind(this);
+    this.modalRef = React.createRef();
+    this.formRef = null;
+  }
+
+  handleUserInput(e) {
+    const name = e.target.name;
+    const value = e.target.value;
+    this.setState({ [name]: value.trim() });
+  }
+
+  onCancel() {
+    this.formRef.reset();
+  }
+
+  submitForm(e) {
+    e.preventDefault();
+    if (this.validator.allValid()) {
+      var { about } = this.state;
+      const data = {
+        about
+      };
+      API.aboutUser(data, result => {
+        if (result.status === "200") {
+          auth.setItem("about", result.data);
+          this.props.closeModal();
+          this.modalRef.remove();
+        } else if (
+          result.status === "404" ||
+          result.status === "403" ||
+          result.status === "500" ||
+          result.status === "400"
+        ) {
+          addNotification(
+            this.notificationDOMRef,
+            "Error",
+            "danger",
+            result.message
+          );
+        } else {
+          addNotification(
+            this.notificationDOMRef,
+            "Error",
+            "warning",
+            "Somgthing went wrong"
+          );
+        }
+      }).catch = error => {
+        addNotification(this.notificationDOMRef, "Error", "warning", error);
+      };
+    } else {
+      this.validator.showMessages();
+      // rerender to show messages for the first time
+      this.forceUpdate();
+    }
+  }
+
+  render() {
+    return (
+      <div
```

Comment:
i will prefer that in one line for better following the flow of the application code. @ayousaf11 what you say?

Reply:
as discussed with @abdulhaqshah 
@Humnaa you can leave this one. Thanks!

Reply:
As we have discussed on slack